docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:
+	* _Seafang Empowered_: Campaign movement range +30% (factionwide), enables teleportation for Wulfrik from port cities he owns (possible?)
+_Reward flavour text:_ Seafang's figurehead quickly absorbed the blood of the slain challengers that were smeared upon it. The ship now glows with eldritch power like never before.
+
+* **Route II - The World Walker**
+  * After a drunken boast made in sight of the Dark Gods, Wulfrik was cursed to forever wander the earth and seek out the fiercest challengers and most monstrous creatures and slay them in single combat.
+  * Occupy at least 5 of the following settlements:
+	* Couronne, Miragliano, Karaz-a-Karak, Altdorf, Castle Drakenhof, Black Crag, Kislev, Naggarond, Lothern, Hexoatl, Skavenblight, Khemri, The Awakening
+  * **Reward**:
+	* _The Eternal Challenger_: Enemy leadership -%10, enemy units near Wulfrik periodically Rampage (attacks the closest enemy and ignores any orders given).

```

Comment:
Love the rewards! Enemy leadership is going o be hard to code tho, if we only want to give it to Wulfrik. A -X leadership in area for characters or something like that would work better IMO.

Reply:
What about giving Wulfrik a taunt spell that drops the target's armour by -60 and makes them rampage?

Flavour text for spell:
"Face me if you dare, stunted whelp, or do you lack even an Elven maid's courage? I thought the Sons of Grungni were great warriors, but perhaps you are no true Dwarf. Indeed, maybe you are instead some breed of bearded goblin, though in truth, I have seen a finer beard on a Troll's back-side."